This does not seem to be a real bug, so do not report it until it gets investigated.

From 6899cc1e2aaa2b1802a719f69a2317ee287f51ff Mon Sep 17 00:00:00 2001
From: Denis Barbier <bouzim@gmail.com>
Date: Wed, 20 Jul 2011 23:01:25 +0000
Subject: [PATCH 16/26] Bugfix: data->xname is freed, do not use a constant
 string but strdup it

---
 src/OpenGl/OpenGl_graduatedtrihedron.cxx |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

Index: oce/src/OpenGl/OpenGl_graduatedtrihedron.cxx
===================================================================
--- oce.orig/src/OpenGl/OpenGl_graduatedtrihedron.cxx
+++ oce/src/OpenGl/OpenGl_graduatedtrihedron.cxx
@@ -451,9 +451,9 @@
     if (index == -1)
     {
         /* Default values */
-        data->xname = "X\0";
-        data->yname = "Y\0"; 
-        data->zname = "Z\0";
+        data->xname = strdup("X\0");
+        data->yname = strdup("Y\0"); 
+        data->zname = strdup("Z\0");
         data->xdrawname = 1;
         data->ydrawname = 1; 
         data->zdrawname = 1;
@@ -1245,4 +1245,4 @@
     ymax = yMax;
     zmax = zMax;
     return TSuccess;
-}
\ No newline at end of file
+}
